code/_onclick/hud/plane_master.dm

```diff
@@ -46,7 +46,7 @@
 /atom/movable/screen/plane_master/game_world
 	name = "game world plane master"
 	plane = GAME_PLANE
-	appearance_flags = PLANE_MASTER //should use client color
+	appearance_flags = PLANE_MASTER|NO_CLIENT_COLOR //should use client color
```

Comment:
The comment says it should use client color, why are adding the no client color flag?

Reply:
This flag is fucked, and apparently inverted.